web/api/swagger.yaml

```diff
@@ -852,8 +852,8 @@ paths:
             properties:
               action:
                 type: string
-                description: The `change:userTimezone` requires to provide also a `timezone` value in the request. The `change:password` requires to provide also a `newPassword` and `oldPassword` in the request.
-                enum: [change:clientsRegistrationEnabled, change:ioDevicesRegistrationEnabled, change:userTimezone, change:password, agree:rules, agree:cookies]
+                description: The `change:userTimezone` requires to provide also a `timezone` value in the request. The `change:password` requires to provide also a `newPassword` and `oldPassword` in the request. The `change:locale` requires to provide also a `locale` value in the request
```

Comment:
ten plik jest wygenerowany z https://app.swaggerhub.com/apis/supla/supla-cloud-api/2.2.0 - jeśli chcesz żeby ta zmiana została, musisz się zarejestrować tam i dopisać ;-)

Reply:
trudno, to nie będzie ;-)